guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -133,21 +133,13 @@ You can help improve the Rails guides by making them more coherent, consistent o
 You can either open a pull request to [Rails](http://github.com/rails/rails) or
 ask the [Rails core team](http://rubyonrails.org/core) for commit access on
 docrails if you contribute regularly.
-Please do not open pull requests in docrails, if you'd like to get feedback on your
```

Comment:
I think we can keep everything about docrails in this file since we talk about giving commit access to docrails.